web/gui/dashboard_info.js

```diff
@@ -4274,6 +4274,47 @@ netdataDashboard.context = {
         info: 'Rate at which segments are sent, including those on current connections, but excluding those containing only retransmitted bytes.'
     },
 
+    // ------------------------------------------------------------------------
+    // WMI (IIS)
+
+    'wmi.iis_website_traffic': {
+        info: 'The total bytes of data that have been sent and received by the web service since the service started.'
+    },
+    'wmi.iis_website_requests_rate': {
+        info: 'The number of requests that have been made since the web service was started.'
+    },
+    'wmi.iis_website_active_connections_count': {
+        info: 'The number of active connections to the web service.'
+    },
+    'wmi.iis_website_users_count': {
+        info: 'The number of users who currently have a request pending with the web service.'
+    },
+    'wmi.iis_website_connection_attempts_rate': {
+        info: 'The number of connections to the web service that have been attempted since the service started.'
+    },
+    'wmi.iis_website_isapi_extension_requests_count': {
+        info: 'The number of ISAPI extension requests that are being processed simultaneously by the web service.'
+    },
+    'wmi.iis_website_ftp_file_transfer_rate': {
```

Comment:
If this is the total number of files transferred why is the metric called ftp file transfer "rate"

Reply:
> We kept name used by windows exporter

You can just copy the description of counters. They are exported as counters, we show them as rates.

Reply:
I copied them @ilyam8 and I also added `per second` to remove possible confusions.

Reply:
For my clarity.

When we convert a cumulative count to a rate, is the rate indicative of a cumulative average OR is it an instantaneous rate?

Reply:
If it is instantaneous rate then we should remove the part of the sentence that says "since the service started". 

Applies to other metrics that use rate as well.

Reply:
I kept the sentence only for the chart it makes sense.